examples/deck.gl/pointcloud/app.js

```diff
@@ -113,25 +138,31 @@ export default class App extends PureComponent {
         // in which case the vertex count is not correct for display as points
         // Proposal: Consider adding a `mesh.points` or `mesh.pointcloud` option to mesh loaders
         // in which case the loader throws away indices and just return the vertices?
-        pointsCount: attributes.POSITION.value.length / 3, // header.vertexCount,
-        points: attributes.POSITION.value,
+        pointsCount: attributes.POSITION.value.length / attributes.POSITION.size,
+        pointsAttribute: attributes.POSITION || null,
+        colorsAttribute: attributes.COLOR_0 || null,
         viewState
       },
       this._rotateCamera
     );
   }
 
   _renderLayers() {
-    const {pointsCount, points, selectedExample} = this.state;
+    const {pointsCount, pointsAttribute, colorsAttribute, selectedExample} = this.state;
 
     return [
-      points &&
+      pointsAttribute &&
         new PointCloudLayer({
           // Layers can't reinitialize with new binary data
           id: `point-cloud-layer-${selectedExample}`,
           coordinateSystem: COORDINATE_SYSTEM.IDENTITY,
           numInstances: pointsCount,
-          instancePositions: points,
+          data: {
+            attributes: {
```

Comment:
This is where you could pass in the output of the `convertToDeckPointCloudData` function, and possibly drop `numInstance`.

Reply:
yep, I realised that the data object can have a length attribute which should accomplish the same thing.